solutions/swb-ui/README.md

```diff
@@ -14,23 +14,15 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ### Prerequisite:
 
-1. Make sure to Follow instructions in [here](../swb-reference/SETUP_v2p1.md##installation) to setup installation of API.
+1. Follow instructions [here](../swb-reference/SETUP_v2p1.md##installation) to setup installation of API.
 
 ### Deploy static website
 
 Project swb-ui can be deployed as a static website using S3 Bucket and CloudFront by following the next steps:
```

Comment:
Nitpick
```suggestion
Project `swb-ui` can be deployed as a static website using S3 Bucket and CloudFront by following the next steps:
```

Reply:
documentation updated to reflect this change